### ISSUE TITLE
Bot Crashing

### ISSUE THREAD

**Issue by tanndlin** [cs-discord-at-ucf/lion#479]
![image](https://user-images.githubusercontent.com/39809146/121994089-43a44a80-cd6a-11eb-8fc8-feb66649de98.png)
```
Jun 14 18:56:25 SandboxHost-637590314747271247 default  info {"status":"fulfillCommand","pluginName":"Coin Toss Plugin","args":[],"user":"SavvyDolphin-77#4717"} 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info     { 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info       message: '{"status":"fulfillCommand","pluginName":"Coin Toss Plugin","args":[],"user":"SavvyDolphin-77#4717"}', 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info       level: 'info', 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info       timestamp: '2021-06-15T01:56:03.923Z', 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info       [Symbol(level)]: 'info', 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info       [Symbol(message)]: '{"message":"{\\"status\\":\\"fulfillCommand\\",\\"pluginName\\":\\"Coin Toss Plugin\\",\\"args\\":[],\\"user\\":\\"SavvyDolphin-77#4717\\"}","level":"info","timestamp":"2021-06-15T01:56:03.923Z"}' 
Jun 14 18:56:25 SandboxHost-637590314747271247 default info     } 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info Loading and running Bot... 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info     { 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info       message: 'Loading and running Bot...', 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info       level: 'info', 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info       timestamp: '2021-06-15T01:56:21.765Z', 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info       [Symbol(level)]: 'info', 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info       [Symbol(message)]: '{"message":"Loading and running Bot...","level":"info","timestamp":"2021-06-15T01:56:21.765Z"}' 
Jun 14 18:56:43 SandboxHost-637590314747271247 default info     } 
```
Last plugin was fulfilled before the crash. It has restarted since last deployment

**Comment by tanndlin** [comment]
still crashing

**Comment by tanndlin** [comment]
I'm sure it has to do with C4
The bot hasnt crashed since restart until someone started playing C4
And it did it twice within half an hour. Both times during a C4 game
@TylerMathis

**Comment by tanndlin** [comment]
> I'm sure it has to do with C4
> The bot hasnt crashed since restart until someone started playing C4
> And it did it twice within half an hour. Both times during a C4 game
> @TylerMathis

Bot still crashed after disabling C4, so its no the culprit

**Comment by tanndlin** [comment]
Bot crashes when mods delete his fuzzy search message from `register` Thanks to Bread for doing that :)

**Comment by suneettipirneni** [comment]
woot! the root cause has been found!

**Comment by tanndlin** [comment]
possibly fixed by [this commit](https://github.com/cs-discord-at-ucf/lion/commit/df4d481402a683e65d40c04893cd4c8e2b6b547f)
will come back later to see if it still crashes

**Comment by tanndlin** [comment]
fixed by #527
